lib/install/management/store_files.go

```diff
@@ -174,12 +172,10 @@ func (d *Dispatcher) deleteFilesIteratively(m *object.FileManager, ds *object.Da
 	return d.deleteVMFSFiles(m, ds, dsPath)
 }
 
-func (d *Dispatcher) deleteVMFSFiles(m *object.FileManager, ds *object.Datastore, dsPath string) error {
+func (d *Dispatcher) deleteVMFSFiles(m *object.DatastoreFileManager, ds *object.Datastore, dsPath string) error {
 	defer trace.End(trace.Begin(dsPath))
 
-	if _, err := tasks.WaitForResult(d.ctx, func(ctx context.Context) (tasks.Task, error) {
-		return m.DeleteDatastoreFile(ctx, dsPath, d.session.Datacenter)
-	}); err != nil {
+	if err := m.Delete(d.ctx, dsPath); err != nil {
```

Comment:
We retry TaskInProgress faults elsewhere, but that isn't thrown by DeleteVirtualDisk_Task or DeleteDatastoreFile_Task: http://pubs.vmware.com/vsphere-65/index.jsp#com.vmware.wssdk.apiref.doc/vim.fault.TaskInProgress.html

I don't see any faults we should be retrying for those methods and the caller(s) of this code already handle the FileNotFound case.